apps/bfd-server/bfd-server-war/pom.xml

```diff
@@ -15,21 +15,6 @@
 		The primary WAR for the BFD Server project. Provides a FHIR API.
 	</description>
 
-	<properties>
-		<datasource-proxy.version>1.9</datasource-proxy.version>
-		<jersey.version>2.25.1</jersey.version>
-
-		<!-- Configure the BFD Server, as it will be run via the exec
-			plugin. These settings are pulled out as POM properties so that they can
-			be adjusted via profiles. -->
-		<its.bfdServer.jvmargs>-Xmx4g</its.bfdServer.jvmargs>
-
-		<!-- Should work out of the box on Linux, but likely needs to be customized
-			(in settings.xml) for Windows dev environments. Seedev/devenv-readme.md
-			for details. -->
-		<bash.exe>bash</bash.exe>
```

Comment:
These two non-version items can stay in bfd-server-war's pom IMO. These are specific to this project and not versions, they are more like server vars.

Reply:
Done